static/docs/commands-reference/status.md

```diff
@@ -7,8 +7,8 @@ cache and remote cache.
 ## Synopsis
 
 ```usage
-usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c]
-                  [-r REMOTE] [-a] [-T] [-d]
+usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums]
```

Comment:
please, address the @jorgeorpinel 's comment ... no split is required

Reply:
Exact txt below 🙂 

```
usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c]
                  [-r REMOTE] [-a] [-T] [-d] [targets [targets ...]]
```

Reply:
reverted it to the original format, unsplit the list. Please review it.